qt/aqt/editor.py

```diff
@@ -79,8 +79,10 @@
 _html = """
 <div id="fields"></div>
 <div id="dupes" class="is-inactive">
-    <a href="#" onclick="pycmd('dupes');return false;">%s</a>
+    <a href="#" onclick="pycmd('dupes');return false;">{}</a>
 </div>
+<div id="not-a-cloze-notetype" class="is-inactive">{}</div>
```

Comment:
Since the error is an enum and can only represent one value at once, do we gain anything from having a separate div for each warning? Are you intending them to be styled differently?